libs/coin-modules/coin-near/src/api/node.ts

```diff
@@ -265,17 +304,18 @@ export const getValidators = makeLRUCache(
 );
 
 export const getCommission = makeLRUCache(
-  async (address: string): Promise<number | null> => {
+  async (validatorAddress: string): Promise<number | null> => {
+    const currencyConfig = getCoinConfig();
     const { data } = await network({
       method: "POST",
-      url: getEnv("API_NEAR_ARCHIVE_NODE"),
+      url: currencyConfig.infra.API_NEAR_PUBLIC_NODE,
       data: {
         jsonrpc: "2.0",
         id: "id",
         method: "query",
         params: {
```

Comment:
Now using Public endpoint for data unrelated to Near account (no sensitive data)